nixos/tests/web-servers/unit-php.nix

```diff
@@ -0,0 +1,97 @@
+import ../make-test-python.nix ({pkgs, ...}:
+ let
+    testdir = pkgs.writeTextDir "www/info.php" "<?php phpinfo();";
+
+in {
+  name = "unit-php-test";
+  meta.maintainers = with pkgs.stdenv.lib.maintainers; [ izorkin ];
+
+  machine = { config, lib, pkgs, ... }: {
+    nixpkgs.overlays = [
+      (self: super: {
+        unit-test = pkgs.unit.override {
+          withPHP72 = true;
```

Comment:
Do you expect a lot of breakages between different php versions? This compiles a unit version just for test itself. In other words we test a unit variant that we is not in nixpkgs itself. Apart from that it looks good.

Reply:
This test checks the simultaneous working of several versions of php. 
Is it possible to enable all php versions by default?
```
diff --git a/pkgs/servers/http/unit/default.nix b/pkgs/servers/http/unit/default.nix
index 407882f176e..57829711e6e 100644
--- a/pkgs/servers/http/unit/default.nix
+++ b/pkgs/servers/http/unit/default.nix
@@ -1,8 +1,8 @@
 { stdenv, fetchFromGitHub, which
 , withPython2 ? false, python2
 , withPython3 ? true, python3, ncurses
-, withPHP72 ? false, php72
-, withPHP73 ? false, php73
+, withPHP72 ? true, php72
+, withPHP73 ? true, php73
 , withPHP74 ? true, php74
 , withPerl528 ? false, perl528
 , withPerl530 ? true, perl530

```

Reply:
But do you think we need to test if it is possible to enable multiple php version?

Reply:
Yes, it’s available to switch between versions without rebuilding the package. Available for testing scripts in different php versions